docs/sources/flow/reference/components/loki.process.md

```diff
@@ -1456,6 +1459,39 @@ The extracted data from the IP used in this example:
 - geoip_autonomous_system_organization: GOOGLE-CLOUD-PLATFORM
 
 
+#### GeoIP with custom fields example
+
+If the MMDB file used is enriched with custom data, e.g. private IP addresses as explained in [the Maxmind blog post](https://github.com/maxmind/mmdb-from-go-blogpost), then it can be extracted from the record using the `custom_lookups` attribute.
```

Comment:
```suggestion
If the MMDB file used is enriched with custom data, for example, private IP addresses as explained in [the Maxmind blog post](https://github.com/maxmind/mmdb-from-go-blogpost), then it can be extracted from the record using the `custom_lookups` attribute.
```